src/nifreeze/model/dmri.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, dataset: DWI, max_b: float | int | None = None, **kwargs):
                 f"DWI dataset is too small ({dataset.gradients.shape[0]} directions)."
             )
 
-        if max_b is not None and max_b > B_MIN:
+        if max_b is not None and max_b > DEFAULT_LOWB_THRESHOLD:
```

Comment:
Note that as explained in https://github.com/nipreps/nifreeze/pull/208/commits/cd9f6cabed30e8119480c47b2da1231b55d14361, the S0 lower bound is changed from 1e-6 to 1e-5.